hack/run-functests.sh

```diff
@@ -23,6 +23,21 @@ export LATENCY_TEST_RUN=${LATENCY_TEST_RUN:-false}
 
 export IS_OPENSHIFT="${IS_OPENSHIFT:-true}"
 
+# Map for the tests paths
+declare -A TESTS_PATHS=\
```

Comment:
can we put this in the common file so it will be easy to find?

Reply:
done